roles/ceph-facts/tasks/facts.yml

```diff
@@ -321,4 +321,4 @@
 
 - name: set_fact use_new_ceph_iscsi package or old ceph-iscsi-config/cli
   set_fact:
-    use_new_ceph_iscsi: "{{ (gateway_ip_list  == '0.0.0.0' and gateway_iqn | length == 0 and client_connections | length == 0 and rbd_devices | length == 0) | bool | ternary(true, false) }}"
\ No newline at end of file
+    use_new_ceph_iscsi: "{{ (gateway_ip_list  == '0.0.0.0' and gateway_iqn | length == 0 and client_connections | length == 0 and rbd_devices | length == 0) | bool | ternary(true, false) }}"
```

Comment:
I've noticed all your recent patches modify the last line, not sure whether it comes from your editor?
could you revert this change?

Reply:
Yes, but should there not be a newline at the end of the last line? IMHO this is a lot cleaner. Try to `cat` a file without a newline at the end... :-)

But of course I can revert this, if you wish.

Reply:
> Yes, but should there not be a newline at the end of the last line? IMHO this is a lot cleaner. Try to `cat` a file without a newline at the end... :-)

you are probably right

> But of course I can revert this, if you wish.

I would prefer to get a new PR especially for fixing this in that case, like a big commit fixing it everywhere :+1: 

perhaps adding a new job in the CI checking for last line deletion could be an idea as well.